platforms/chibios/drivers/vendor/RP/RP2040/ws2812_vendor.c

```diff
@@ -161,7 +164,7 @@ static void ws2812_dma_callback(void* p, uint32_t ct) {
     // Convert from ns to us
     time_to_completion /= 1000;
 
-    LAST_TRANSFER = chSysGetRealtimeCounterX() + time_to_completion + WS2812_TRST_US;
+    LAST_TRANSFER = time_us_64() + time_to_completion + WS2812_TRST_US;
```

Comment:
Thanks, it is better to use the intended api for the type. Using `make_timeout_time_us()` would be even better but I'm in favor of keeping the additional includes on the pico-sdk to a minimum to prevent further naming conflicts and to keep the compile time down.